### DISCUSSION TITLE
PubSub Lite

### DISCUSSION THREAD

**Discussion by n1o** [googleapis/nodejs-pubsub#1445] [Q&A]
Hi,

I would like to work with pub/sub lite topics, however I am having a hard time to find any mention how to use them from nodejs. Is this library the right place?

**Comment by hongalex** [comment]
Hi, this is fairly late but per https://github.com/googleapis/nodejs-pubsub/issues/1612 we're not looking to create a Pub/Sub Lite library for node at this point.
